### PR TITLE
Anti-Island Member Kick

### PR DESCRIPTION
**Pull request by Oribuin**
The plugin will no longer kick island members and send them back to spawn when the island is set to private

### CODE REVIEW

File: src/main/java/com/iridium/iridiumskyblock/commands/PrivateCommand.java [hunk 1]
```diff
@@ -45,6 +45,10 @@ public boolean execute(CommandSender sender, String[] args) {
             int visitorCount = 0;
             List<IslandTrusted> islandTrusted = IridiumSkyblock.getInstance().getDatabaseManager().getIslandTrustedTableManager().getEntries(island.get());
             for (User visitor : IridiumSkyblock.getInstance().getIslandManager().getPlayersOnIsland(island.get())) {
+
+                if (island.get().getMembers().contains(visitor))
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/commands/PrivateCommand.java:49]: Island#getMembers should be saved as a variable somewhere to be more efficient, it wont make a huge difference but like ye

Comment (MertUnverdi) [reply src/main/java/com/iridium/iridiumskyblock/commands/PrivateCommand.java:49]: This already checks below line?

Comment (PeachesMLG) [reply src/main/java/com/iridium/iridiumskyblock/commands/PrivateCommand.java:49]: no island.getMembers performs a binary search on the users entries to get the members, so we should be saving something like Lisit<User> islandMembers = island.get().getMemebers() to make it more efficient

File: src/main/java/com/iridium/iridiumskyblock/commands/PrivateCommand.java [hunk 2]
```diff
@@ -44,7 +44,13 @@ public boolean execute(CommandSender sender, String[] args) {
...
             int visitorCount = 0;
             List<IslandTrusted> islandTrusted = IridiumSkyblock.getInstance().getDatabaseManager().getIslandTrustedTableManager().getEntries(island.get());
+            final List<User> islandMembers = island.get().getMembers();
+
             for (User visitor : IridiumSkyblock.getInstance().getIslandManager().getPlayersOnIsland(island.get())) {
+
+                if (islandMembers.contains(visitor))
+                    continue;
+
                 if (visitor.getIsland().map(Island::getId).orElse(0) == island.get().getId() || islandTrusted.stream().anyMatch(trustedIsland -> trustedIsland.getUser().getUuid() == visitor.getUuid())) {
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/commands/PrivateCommand.java:54]: Slash just pointed out to me, we already check this here :eyes:

**Pull request merged**